Libplanet/Tx/TxViolatingBlockPolicyException.cs

```diff
@@ -1,26 +0,0 @@
-#nullable enable
-using System;
-
-namespace Libplanet.Tx
-{
-    /// <summary>
-    /// The exception that is thrown when a <see cref="Transaction{T}"/> does not follow the
-    /// block policy.
-    /// </summary>
-    [Serializable]
-    public class TxViolatingBlockPolicyException : InvalidTxException
```

Comment:
This removal has to be recorded in the changelog.

Reply:
Done. See 81532875ddca91a6d122800aba0704ef63fb03d3.